packages/main/src/Popup.js

```diff
@@ -278,10 +278,17 @@ class Popup extends UI5Element {
 	}
 
 	_onfocusout(e) {
-		// relatedTarget is the element, which will get focus. If no such element exists, focus the root
+		// relatedTarget is the element, which will get focus. If no such element exists, focus the root.
```

Comment:
... focus the root. This happens after the mouse is released to allow text selection.

Reply:
Done